client/data/disputes/actions.js

```diff
@@ -30,17 +32,62 @@ export function updateDisputes( query, data ) {
 	};
 }
 
+export function updateIsSavingEvidenceForDispute(
+	disputeId,
+	isSavingEvidence
+) {
+	return {
+		type: TYPES.SET_IS_SAVING_EVIDENCE_FOR_DISPUTE,
+		data: {
+			isSavingEvidence,
+			id: disputeId,
+		},
+	};
+}
+
+export function updateEvidenceTransientForDispute(
+	disputeId,
+	evidenceTransient
+) {
+	return {
+		type: TYPES.SET_EVIDENCE_TRANSIENT_FOR_DISPUTE,
+		data: { id: disputeId, evidenceTransient },
+	};
+}
+
+export function updateIsUploadingEvidenceForDispute(
+	disputeId,
+	key,
+	isUploadingEvidenceForDispute
+) {
+	return {
+		type: TYPES.SET_IS_UPLOADING_EVIDENCE_FOR_DISPUTE,
+		data: { id: disputeId, key, isUploadingEvidenceForDispute },
+	};
+}
+
+export function updateEvidenceUploadErrorsForDispute(
+	disputeId,
+	key,
+	errorMessage
+) {
+	return {
+		type: TYPES.SET_EVIDENCE_UPLOAD_ERRORS_FOR_DISPUTE,
+		data: { id: disputeId, key, errorMessage },
+	};
+}
+
 export function* acceptDispute( id ) {
 	try {
-		yield dispatch( STORE_NAME, 'startResolution', 'getDispute', [ id ] );
+		yield dispatch( STORE_NAME ).startResolution( 'getDispute', [ id ] );
```

Comment:
@dechov using the non-deprecated `dispatch` function here broke the unit tests and I'm not sure why or how to fix the issue. Do you think you could take a look? I think you're the one that wrote the initial tests 🤔

Reply:
Seems the previous `dispatch` was simpler since it was a single call that just returned an object – we may need to do the sort of thing done for the settings tests [here](https://github.com/Automattic/woocommerce-payments/blob/958893ec35e215e7ab8736ca12439ee4068a7911/client/data/settings/test/actions.js#L18-L35).

Reply:
Yeah, something similar, although I had to massage the code a bit since the dispute retrieved via `apiFetch` in `acceptDispute` was always `undefined` when I ran the test, even though I set the return value from `apiFetch` to the `mockDispute` object.

It was a trivial thing though, just change `const message = dispute.order // ...` to `const message = dispute?.order //...`.

Reply:
I fixed this in 4e9358a9a6d79b89297eb5cdb01d9dc4191eb1a9